src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -167,57 +173,100 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseSer
         return ResponseEntity.ok(responseComplaints);
     }
 
-    @GetMapping("/complaints")
+    /**
+     * Get /exercises/:exerciseId/more-feedback-for-tutor-dashboard
+     * <p>
+     * Get all the more feedback requests associated to an exercise, that are about the tutor who is doing the request.
+     * @param exerciseId the id of the exercise we are interested in
+     * @return the ResponseEntity with status 200 (OK) and a list of more feedback requests. The list can be empty
+     */
+    @GetMapping("/exercises/{exerciseId}/more-feedback-for-tutor-dashboard")
     @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
-    public ResponseEntity<List<Complaint>> getComplaintsFilteredBy(@RequestParam(required = false) Long tutorId, @RequestParam(required = false) Long exerciseId,
-            @RequestParam(required = false) Long courseId) {
-        if (tutorId == null && exerciseId == null && courseId == null) {
-            throw new BadRequestAlertException("You need to specify at least one between tutorId, exerciseId, and courseId", ENTITY_NAME, "specifyFilter");
+    public ResponseEntity<List<Complaint>> getMoreFeedbackRequestsForTutorDashboard(@PathVariable Long exerciseId, Principal principal) {
+        Exercise exercise = exerciseService.findOne(exerciseId);
+        if (!authCheckService.isAtLeastTeachingAssistantForExercise(exercise)) {
+            return forbidden();
         }
 
+        List<Complaint> responseComplaints = complaintService.getMyMoreFeedbackRequests(exerciseId, principal);
+        return ResponseEntity.ok(responseComplaints);
+    }
+
+    /**
+     * Get /complaints
+     * <p>
+     * Get all the complaints for tutor.
+     * @param complaintType the type of complaints we are interested in
+     * @return the ResponseEntity with status 200 (OK) and a list of complaints. The list can be empty
+     */
+    @GetMapping("/complaints")
+    @PreAuthorize("hasAnyRole('TA', 'ADMIN')")
```

Comment:
Why do you leave out instructors? They should NEVER have less rights than tutors.
And instructors can assess as well, so there could be complaints about them as well
(They can also use the tutor course dashboard)